projects/observability/src/shared/dashboard/data/graphql/topology/topology-data-source.model.ts

```diff
@@ -160,3 +214,13 @@ export interface TopologyData {
   nodeMetrics: TopologyMetricsData;
   edgeMetrics: TopologyMetricsData;
 }
+
+export interface TopologyEdgeFilterConfig {
+  entityType: string;
+  fields: string[];
+}
+
+interface TopologyFilters {
+  forNode: GraphQlFieldFilter[];
```

Comment:
could we just call it `nodes` and `edges`? The <for> in the variable name gives it an impression that we are writing a for loop

Reply:
Done